client/templates/globals/profileEditDialog.js

```diff
@@ -31,25 +52,33 @@ Template.profileEditDialog.events({
 
         if (Meteor.user().isDemoUser) { 
             return; 
-        } 
-        
-        let uLongName = tmpl.find('#id_longName').value;
-        let uEmailAddress = tmpl.find('#id_emailAddress').value;
+        }
 
-        tmpl.$('#btnEditProfileSave').prop('disabled',true);
+        let uEmailAddress = tmpl.find('#id_emailAddress').value;
 
-        Meteor.call('users.editProfile', Meteor.userId(), uEmailAddress,uLongName, function (error) {
-            if (error) {
-                tmpl.$('#btnEditProfileSave').prop('disabled',false);
-                console.log(error);
-                showError(evt, error);
-            } else {
-                (new FlashMessage('OK', 'Profile edited.', 'alert-success', 2000)).show();
+        if (Meteor.settings.public.sendVerificationEmail) {
+            let changeUserMail = () => {
+                updateUserProfil(tmpl);
```

Comment:
@migerh Thanks for your answer! @RobNeXX could you please check whether meteor queues method calls from the same client. Otherwise we have to make sure that logout is called after `updateUserProfile` for example by making `updateUserProfile` returning a promise object.